api/src/main/java/keywhiz/api/automation/v2/SecretDetailResponseV2.java

```diff
@@ -117,6 +136,7 @@ public SecretDetailResponseV2 build() {
   @Override public final String toString() {
     return MoreObjects.toStringHelper(this)
         .add("name", name())
+        .add("version", version())   // TODO: Should we remove this so it's not as exposed to the user?
```

Comment:
We need the version as part of the SecretDetailResponseV2, since as far as I can tell it's the only easy way to retrieve a particular version.  I'm not sure whether we should put it in the toString method, though.  Having it there could be useful for debugging, but it also exposes that identifier to a user.